test/gnuradio/test_adi_gnuradio.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import pyguit
+import time
+import os
+import shutil
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
+
+
+
+class TestADIGnuradio:
+
+    @classmethod
+    def setup_class(self):
+        '''Setup the pyguit object. Will be called once before the start of test'''
+        self.gui = pyguit.gui()
+        self.gui.attach_openbox()
+        # self.gui.run_ssh_agent()
+        time.sleep(10)
+        try:
+            os.makedirs("results")
+            os.makedirs("logs")
+        except FileExistsError:
+            # directory already exists
+            shutil.rmtree('results')
+            shutil.rmtree('logs')
+            os.makedirs("results")
+            os.makedirs("logs")
+
+         # Initialize XML log file
+        self.gui.log_root = ET.Element("logs")
```

Comment:
Can you show (atleast in comment) an example format of this xml log? When pytest is run, we could actually generate an xml file of the result, but if this xml log has a different purpose, we could add this as well.

Reply:
here is the sample xml file, 
![image](https://github.com/sdgtt/gui-testing/assets/114046109/f9e276ac-1c9c-42b4-b598-c3b9d324828a)

I'll check the logfile we have when running the pytest.